config.js

```diff
@@ -1,6 +1,6 @@
 var config = {
-    port: process.env.PORT || 8080,
+    port: process.env.PORT || 8079,
```

Comment:
Why did you change the port?
```suggestion
    port: process.env.PORT || 8080,
```